server/src/main/java/org/elasticsearch/index/snapshots/IndexShardSnapshotStatus.java

```diff
@@ -191,6 +191,10 @@ public Stage getStage() {
         return stage.get();
     }
 
+    public long getTotalTime() {
```

Comment:
I know this is the name of the field but could we include the unit (millis?) in the name of this getter at least?